comtypes/__init__.py

```diff
@@ -4,7 +4,17 @@
 import atexit
 from ctypes import *
 from ctypes import _SimpleCData
-from _ctypes import COMError
+try:
+    from _ctypes import COMError
+except ImportError as e:
+    e.msg = "\n".join((
```

Comment:
Rewriting the exception message attribute directly is not recommended because the original exception information will be lost.

Raising a new exception while preserving the original exception information is easy with Py3 because the [from clause](https://docs.python.org/3/library/exceptions.html#exception-context) can be used.

```py
…
try:
    from _ctypes import COMError
except ImportError as e:
    msg = …
    raise ImportError(msg) from e
```

Rather than making an effort to achieve the same thing in Py2 for the `master` branch, it would be better to work on the `drop_py2` plan(#392) as it is in progress.

Please change the base of this PR to `drop_py2` and rebase the branch as if it was made from `drop_py2` as well, or remake branch and re-submit PR.